### PR TITLE
chore: init github actions for toolchains & refact submodules

### PR DESCRIPTION
**Pull request by imbajin**
CI/CD actions will do in a separate PR

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 1]
```diff
@@ -11,7 +11,7 @@ body:
         ### Note (特别注意) : 
 
         > 1. 请先**搜索**现有的[Server-Issues](https://github.com/hugegraph/hugegraph/issues) 与
-        [Tools-Issues](https://github.com/hugegraph/hugegraph-tools/issues) 中没有与您相同
+        [Toolchains-Issues](https://github.com/hugegraph/hugegraph-loader/issues) 中没有与您相同
```
Comment (javeme) [inline .github/ISSUE_TEMPLATE/bug_report.yml:14]: Toolchains => Toolchain

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 2]
```diff
@@ -26,16 +26,18 @@ body:
...
-        - logic (逻辑设计问题)
-        - others (please edit later)
+        - others (please comment below)
   
   - type: checkboxes
     attributes:
       label: Before submit
       options:
-        - label: 我已经确认现有的 [Server-Issues](https://github.com/hugegraph/hugegraph/issues) 与 [Tools-Issues](https://github.com/hugegraph/hugegraph-tools/issues) 中没有相同 / 重复问题
+        - label: 我已经确认现有的 [Server-Issues](https://github.com/hugegraph/hugegraph/issues) 与 [Toolchains-Issues](https://github.com/hugegraph/hugegraph-loader/issues) 中没有相同 / 重复问题
```
Comment (javeme) [inline .github/ISSUE_TEMPLATE/bug_report.yml:40]: ditto

File: .github/workflows/stale.yml [hunk 3]
```diff
@@ -20,17 +20,19 @@ jobs:
...
         operations-per-run: 10
-        start-date: '2018-12-01T00:00:00Z'
+        start-date: '2017-10-01T00:00:00Z'
 
         exempt-all-assignees: true
         remove-stale-when-updated: true
         exempt-all-pr-milestones: true
         delete-branch: false
-        enable-statistics: true
+        enable-statistics: true
```
Comment (javeme) [inline .github/workflows/stale.yml:38]: expect a new line

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 4]
```diff
@@ -26,16 +26,18 @@ body:
...
-        - logic (逻辑设计问题)
-        - others (please edit later)
+        - others (please comment below)
   
   - type: checkboxes
     attributes:
       label: Before submit
       options:
-        - label: 我已经确认现有的 [Server-Issues](https://github.com/hugegraph/hugegraph/issues) 与 [Tools-Issues](https://github.com/hugegraph/hugegraph-tools/issues) 中没有相同 / 重复问题
+        - label: 我已经确认现有的 [Server-Issues](https://github.com/hugegraph/hugegraph/issues) 与 [toolchain-Issues](https://github.com/hugegraph/hugegraph-loader/issues) 中没有相同 / 重复问题
```
Comment (javeme) [inline .github/ISSUE_TEMPLATE/bug_report.yml:40]: Toolchain-Issues

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 5]
```diff
@@ -11,7 +11,7 @@ body:
         ### Note (特别注意) : 
 
         > 1. 请先**搜索**现有的[Server-Issues](https://github.com/hugegraph/hugegraph/issues) 与
-        [Tools-Issues](https://github.com/hugegraph/hugegraph-tools/issues) 中没有与您相同
+        [toolchain-Issues](https://github.com/hugegraph/hugegraph-loader/issues) 中没有与您相同
```
Comment (javeme) [inline .github/ISSUE_TEMPLATE/bug_report.yml:14]: Toolchain-Issues

**Review by javeme — APPROVED** [review]

**Comment by imbajin** [comment]
TODO in next PR:
- [ ] image

**Pull request merged**